package.json

```diff
@@ -11,18 +11,19 @@
   "author": "",
   "license": "MIT",
   "devDependencies": {
-    "@rollup/plugin-commonjs": "^15.1.0",
-    "@rollup/plugin-node-resolve": "^9.0.0",
-    "@rollup/plugin-typescript": "^6.0.0",
-    "@types/node": "^14.14.10",
+    "@rollup/plugin-commonjs": "15.1.0",
```

Comment:
Using exact version numbers means all contributers will have the same exact build.